### DISCUSSION TITLE
I am looking for an old working GitHub repository of Discord-AI-Chatbot. Could you please help me find it?

### DISCUSSION THREAD

**Discussion by GameHak** [mishl-dev/Discord-AI-Chatbot#247] [Q&A] [answered]
Hello, I am looking for version of Discord-AI-Chatbot in the GitHub repository with image generation V3 V4 because when I do, it tells me that it generates an image but it does not generate it. Could you tell me which version works? I thank you in advance for your help.

**Comment by mishl-dev** [comment, marked as answer]
Check old commits https://github.com/mishalhossin/Discord-AI-Chatbot/commits/main/
